lib/ohmqtt/ohmqtt.cpp

```diff
@@ -64,11 +75,26 @@ void ohmqtt::netSetup() {
   });
   ArduinoOTA.onError([](ota_error_t error) {
     Serial.printf("Error[%u]: ", error);
-    if      (error == OTA_AUTH_ERROR   ) Serial.println("Auth Failed");
-    else if (error == OTA_BEGIN_ERROR  ) Serial.println("Begin Failed");
-    else if (error == OTA_CONNECT_ERROR) Serial.println("Connect Failed");
-    else if (error == OTA_RECEIVE_ERROR) Serial.println("Receive Failed");
-    else if (error == OTA_END_ERROR    ) Serial.println("End Failed");
+    switch (error) {
```

Comment:
This looks gross but it is actually really neat as we save three instructions per if statement (which is noticeable when running code on IoT devices).